components/raftstore/src/coprocessor/config.rs

```diff
@@ -87,6 +90,7 @@ impl Default for Config {
             enable_region_bucket: false,
             region_bucket_size: DEFAULT_BUCKET_SIZE,
             region_size_threshold_for_approximate: DEFAULT_LARGE_REGION_SIZE_THRESHOLD,
+            region_bucket_merge_size: DEFAULT_BUCKET_MERGE_SIZE,
```

Comment:
Suggest that the default value of it is better to be calculated from `region_bucket_size` e.g. `region_bucket_size / 2`, otherwise it would be painful to tune the config.

Reply:
Do you mean region_bucket_merge_size's default value is based on actual value of region_bucket_size (not the default DEFAULT_BUCKET_SIZE) right?
But apparently it cannot be done in default() because all the values here are hardcoded.